internal/olm/manager.go

```diff
@@ -62,6 +66,7 @@ func (m *Manager) initialize() (err error) {
 			m.Timeout = DefaultTimeout
 		}
 	})
```

Comment:
This will block forever unless you pass a nil error at the end of `func()`
```suggestion
	errChan <- nil
	})
```

Reply:
Oh, yes, my mistake, thanks.